cmd/internal/ecs/add.go

```diff
@@ -571,10 +589,38 @@ func getServiceState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowS
 	return awf_next(confirmState)
```

Comment:
```suggestion
	return awf_next(getSecretState)
```

Reply:
I am very confused as to how this worked when I tested it.

... oh, because they both default to false.